ding/utils/plot.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import seaborn as sns
```

Comment:
add seaborn in `setup.py`, and remember it whenever you import some new packages